modular_skyrat/code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -0,0 +1,225 @@
+	/obj/item/hemostat/ = 10,
+	/obj/item/hot_potato/harmless/toy = 5,
+	/obj/item/laser_pointer/red = 10,
+	/obj/item/latexballon = 5,
+	/obj/item/light/bulb = 5,
+	/obj/item/light/bulb/broken = 5,
+	/obj/item/light/tube = 5,
+	/obj/item/light/tube/broken = 5,
+	/obj/item/lighter = 5,
+	/obj/item/megaphone/sec = 1,
+	/obj/item/mining_scanner = 2,
+	/obj/item/mop = 5,
+	/obj/item/multitool = 25,
+	/obj/item/organ/tail/cat = 1,
+	/obj/item/organ/tail/lizard = 1,
+	/obj/item/paint/paint_remover = 5,
+	/obj/item/paper_bin = 10,
+	/obj/item/pda = 5,
+	/obj/item/pen/fountain = 5,
+	/obj/item/pickaxe = 25,
+	/obj/item/plant_analyzer = 5,
+	/obj/item/radio = 100,
+	/obj/item/reagent_containers/food/condiment/mayonnaise = 5,
+	/obj/item/reagent_containers/glass/beaker = 50,
+	/obj/item/reagent_containers/glass/beaker/large = 25,
+	/obj/item/reagent_containers/glass/bucket = 25,
+	/obj/item/reagent_containers/pill/floorpill = 100,
+	/obj/item/reagent_containers/rag = 10,
+	/obj/item/reagent_containers/spray/cleaner = 25,
+	/obj/item/reagent_containers/spray/spraytan = 5,
+	/obj/item/restraints/handcuffs = 5,
+	/obj/item/restraints/handcuffs/fake/kinky = 5,
+	/obj/item/restraints/legcuffs/bola = 5,
+	/obj/item/retractor = 25,
+	/obj/item/roller = 5,
+	/obj/item/scalpel = 25,
+	/obj/item/shard = 200,
+	/obj/item/shovel = 25,
+	/obj/item/skub = 25,
+	/obj/item/slime_scanner = 5,
+	/obj/item/soap = 25,
+	/obj/item/stack/conveyor/fifteen = 25,
+	/obj/item/stack/marker_beacon/ten = 10,
+	/obj/item/stack/medical/bruise_pack = 50,
+	/obj/item/stack/medical/gauze = 25,
+	/obj/item/stack/medical/gauze/improvised = 75,
+	/obj/item/stack/medical/ointment = 50,
+	/obj/item/stack/rods/twentyfive = 25,
+	/obj/item/stack/rods/fifty = 25,
+	/obj/item/stack/rods/ten = 75,
+	/obj/item/stack/sheet/cardboard/fifty = 25,
+	/obj/item/stack/sheet/cloth/thirty = 25,
+	/obj/item/stack/sheet/cotton/thirty = 25,
+	/obj/item/stack/sheet/glass/fifty = 25,
+	/obj/item/stack/sheet/glass/five = 75,
+	/obj/item/stack/sheet/metal/fifty = 25,
+	/obj/item/stack/sheet/metal/five = 50,
+	/obj/item/stack/sheet/metal/ten = 50,
+	/obj/item/stack/sheet/metal/twenty = 75,
+	/obj/item/stack/sheet/mineral/bamboo/ten = 25,
+	/obj/item/stack/sheet/mineral/coal/ten = 10,
+	/obj/item/stack/sheet/mineral/plasma = 25,
+	/obj/item/stack/sheet/mineral/wood/twenty = 50,
+	/obj/item/stack/sheet/plasmaglass = 10,
+	/obj/item/stack/sheet/plasteel = 10,
+	/obj/item/stack/sheet/plastic/five = 75,
+	/obj/item/stack/sheet/rglass = 50,
+	/obj/item/stake/basic = 25,
+	/obj/item/stake/hardened = 5,
+	/obj/item/stock_parts/capacitor = 50,
+	/obj/item/stock_parts/cell = 50,
+	/obj/item/stock_parts/cell/upgraded = 100,
+	/obj/item/stock_parts/cell/high = 100,
+	/obj/item/stock_parts/manipulator = 50,
+	/obj/item/stock_parts/matter_bin = 50,
+	/obj/item/stock_parts/micro_laser = 50,
+	/obj/item/stock_parts/scanning_module = 50,
+	/obj/item/storage/belt/utility = 100,
+	/obj/item/storage/belt = 10,
```

Comment:
This is REALLY REALLY strong, like it holds ***anything w_class = normal***

Reply:
I'll drop it to 1% because I recall that it's actually a maint loot option.